### ISSUE TITLE
notify user about job errors on arrival

### ISSUE THREAD

**Issue by yegor256** [zerocracy/baza#133]
Currently, in `pipeline.rb`, we notify the user via `human.notify()` when a job finishes with errors. However, we don't do the same when a job arrives with errors. Let's do this, also in the `pipeline.rb`, before we start processing the `input`.

**Comment by yegor256** [comment]
@Suban05 it's already done
